app/controllers/api/subcollections/tags.rb

```diff
@@ -85,7 +85,7 @@ def parse_tag(data)
 
       def parse_tag_from_href(data)
         href = data["href"]
-        tag  = if href && href.match(%r{^.*/tags/#{CID_OR_ID_MATCHER}$})
+        tag  = if href && href.match(%r{^.*/tags/#{ArRegion::CID_OR_ID_MATCHER}$})
```

Comment:
getting `uninitialized constant Api::Subcollections::Tags::CID_OR_ID_MATCHER` without it in the tests